src/scss/includes/direction.scss

```diff
@@ -740,4 +740,8 @@ button.direction_shortcut {
       }
     }
   }
+
+  .autocomplete_suggestions {
```

Comment:
I see no difference by commenting this rule ?

Reply:
when it's commented, the directions suggest list on mobile doesn't take all the height of the screen as required:

![image](https://user-images.githubusercontent.com/1225909/95206423-f9d09100-07e6-11eb-9ad8-750ccd7697b8.png)

![image](https://user-images.githubusercontent.com/1225909/95206482-09e87080-07e7-11eb-8e66-b7fb39f2a7ea.png)